src/stores/summary-card-store.ts

```diff
@@ -157,11 +157,11 @@ export default class SummaryCardStore {
     onBotContractEvent(contract: TContractInfo) {
         const { profit } = contract;
         const indicative = getIndicativePrice(contract as ProposalOpenContract);
+        this.profit = profit;
 
         if (this.contract_id !== contract.id) {
             this.clear(false);
             this.contract_id = contract.id;
-            this.profit = profit;
```

Comment:
we need the value of each transaction of the profit field relative to one contract, not just the first one